include/ttmlir/Dialect/TTNN/IR/TTNNOpModelInterface.td

```diff
@@ -19,32 +19,25 @@ def TTNN_OpModelInterface : OpInterface<"OpModel"> {
             }],
             /*retTy=*/"size_t",
             /*methodName=*/"getOpPerfCycles",
-            /*args=*/(ins "const std::vector<TTNNLayoutAttr>&":$input_layouts, "const TTNNLayoutAttr&":$output_layout),
+            /*args=*/(ins "const std::vector<TTNNLayoutAttr>&":$inputs, "const TTNNLayoutAttr&":$output),
             /*methodBody=*/"",
             /*defaultImplementation=*/"return std::numeric_limits<size_t>::max();"
         >,
         InterfaceMethod<
             /*desc=*/[{
-                Returns the op memory L1 usage estimate in bytes. The return value is a tuple of 3 values:
-                - The first value is CB L1 peak allocation in bytes.
-                - The second value is Tensor L1 peak allocation in bytes.
-                - The third value is Output L1 buffer allocation in bytes.
+                Returns a tuple of three values:**
+                1. A boolean indicating if the op is legal for the given input/output layouts.
+                2. If the op is legal, a tuple of three values representing the op memory L1 usage estimate in bytes.
+                   - The first value is the CB L1 peak allocation in bytes.
+                   - The second value is the Tensor L1 peak allocation in bytes.
```

Comment:
What's the diff between CB and Tensor?

Reply:
circular buffers are used for communication between riscv cores within tensixes; and for shards. while tensors are higher level concept. they are allocated from different sides of the memory.